scripts/cnn_variant_wdl/cram2model.json

```diff
@@ -0,0 +1,20 @@
+{
+	"Cram2TrainedModel.input_cram": "gs://broad-dsde-methods/cnn-variant-score/bams/NA12878_PLUMBING.cram",
+	"Cram2TrainedModel.reference_fasta": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta",
+	"Cram2TrainedModel.reference_dict": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
+	"Cram2TrainedModel.reference_fasta_index": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.fai",
+	"Cram2TrainedModel.output_prefix": "plumbing_na12878",
+	"Cram2TrainedModel.tensor_type" : "read_tensor",
```

Comment:
Where does the architecture get specified?

Reply:
by default, I will add an optional argument after the tech debt PR which update how the architecture is specified.